packages/chat-widget/src/index.tsx

```diff
@@ -325,6 +326,20 @@ const ConversationHandlerContext = createContext<ConversationHandler | null>(
   null,
 );
 
+// eslint-disable-next-line jsdoc/require-returns
```

Comment:
this eslint-disable-next-line is unecessary if the const isn't exported

Reply:
@michaelglass it wasn't passing...